test/test.1.js

```diff
@@ -1,4 +1,14 @@
-var logger = require('sp-json-logger');
```

Comment:
This worked for me because I relied on `npm link`
I debated if I should be testing it as a pseudo-published module `sp-json-logger` or as a relative file: `./../src/bunyanWrapper` but didn't have strong opinions either way.
I'm fine with this change.